### ISSUE TITLE
p5.SoundRecorder example broken

### ISSUE THREAD

**Issue by davepagurek** [processing/p5.js-sound#715]
Running the example here https://p5js.org/reference/#/p5.SoundRecorder gives me this error in the console:

```
arraySequence[channel] is undefined
```

...which later causes it to try to create a sound file with buffer length 0, which throws a different error.

**Comment by KevinGrajeda** [comment]
The recording of the audio and the wav download works well for me, I tried in firefox and chrome, but it logs this warning:
```
p5.min.js?v=9299cb:2 textAlign(*, CENTER) requires x, y, width and height
```
and the 'Done! Tap to play and download' text does not appear.

**Comment by davepagurek** [comment]
Thanks for helping narrow this down! What OS do you use? in Chrome I get the same result as you, but get errors in Firefox 108 on MacOS 12.5.1.

For the text issue, it seems like that was added as a result of the fix for this issue https://github.com/processing/p5.js/issues/5785. If we want to fix it now, we can change these examples to `text('something', 0, 0, width, height)`, but I'll also mention in the p5 repo that it seems to be causing existing examples to stop working.

**Comment by davepagurek** [comment]
There's now an issue here https://github.com/processing/p5.js/issues/5933 to track that text issue!

**Comment by KevinGrajeda** [comment]
I am using windows 10

**Comment by davepagurek** [comment]
Ok, looks like it might just be a Firefox Mac issue then, which might not be p5.sound's doing. We can keep track of this and check back in the next version in case this is a Firefox Mac regression that gets fixed.

**Comment by tuminzee** [comment]
@davepagurek  I am getting this while using it on Firefox Mac

<img width="1624" alt="image" src="https://user-images.githubusercontent.com/58180803/211169197-be04349b-0eb2-4dad-a963-ae17cf548f78.png">

**Comment by davepagurek** [comment]
Thanks @tuminzee, looks like it is indeed a Firefox for Mac issue and not just for me! I haven't yet looked into what part is going wrong here, but let me know in the mean time if anyone feels like taking a look and gets any insight.

**Comment by tuminzee** [comment]
sure thing @davepagurek  I would love to give it a try, although I am not familiar with the whole codebase but this could be a good opportunity of learning for me, as in future I want to contribute more to this community

**Comment by davepagurek** [comment]
Sounds good! I'll assign this issue to you, let me know if I can give any pointers.

**Comment by tuminzee** [comment]
Thank you @davepagurek 
I'll keep you updated

**Comment by tuminzee** [comment]
I found this https://github.com/processing/p5.js-sound/issues/499#issuecomment-994308373
I will go in this direction

**Comment by davepagurek** [comment]
good catch! that seems like the underlying cause, hopefully fixing that will fix this too!

**Comment by tuminzee** [comment]
@davepagurek I'll dig up the cause. 
I'll keep you updated.
